types/tonic-ui__react-icons/scripts/generate-definitions/tmicon.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@trendmicro/tmicon' {
+  const icons: Array<{ name: string }>;
+}
```

Comment:
The scripts can't contain dts files, they will be published. You'll need to remove this file.